tower/src/util/mod.rs

```diff
@@ -61,6 +61,18 @@ pub mod future {
 /// adapters
 pub trait ServiceExt<Request>: tower_service::Service<Request> {
     /// Yields a mutable reference to the service when it is ready to accept a request.
+    fn ready(&mut self) -> Ready<'_, Self, Request>
+    where
+        Self: Sized,
+    {
+        Ready::new(self)
+    }
+
+    /// Yields a mutable reference to the service when it is ready to accept a request.
+    #[deprecated(
+        since = "0.4.6",
+        note = "Please use the ServiceExt::ready method instead"
```

Comment:
nit: going off the stdlib's example, i think these usually start with a lowercase letter, since they're part of a sentence, include backticks for Rust type names (which renders in the RustDoc). like:
```suggestion
        note = "please use the `ServiceExt::ready` method instead"
```